public/lessons/javascript/input_and_output.md

```diff
@@ -222,8 +224,6 @@ function ask(questionText) {
 }
 ```
 
-> This is called "boilerplate code" -- you don't need to fully understand it before using it.
```

Comment:
Is there somewhere else you explain boilerplate code?

Reply:
Nope